RELEASE_NOTES.md

```diff
@@ -2,6 +2,8 @@
 
 ## New in git-machete 3.10.1
 
+- added: `git machete {diff, log, add, anno, fork-point, is-managed, show}` commands now support both short (`develop`) and full (`refs/heads/develop`) branch names
```

Comment:
Huh right! that's indeed a MINOR-level change in that sense, lol &mdash; a user-facing feature, so it'd be 3.11.0 ...

Reply:
I'd rather leave 3.10.1 (a PATCH-level update i.e. a bugfix)... but change the RELEASE_NOTES entry to
```
- fixed: git machete diff doesn't crash when ...
- fixed: git machete {log, add, anno, fork-point, is-managed, show} doesn't crash when...
```